### ISSUE TITLE
Better highlight tab for bufferline

### ISSUE THREAD

**Issue by huyhoang8398** [bluz71/vim-moonfly-colors#46]
<img width="985" alt="Screenshot 2023-08-01 at 04 16 40" src="https://github.com/bluz71/vim-moonfly-colors/assets/25379882/a5f2802e-9bbe-4c1b-9a6b-37fba6c5454e">
Im currently using bufferline plugin, with option:
`separtor_line = "sloped"` 

Is this possible to make the buffer tab more noticeable? for example changing the background of the line, and keep the color of the buffer tab?
![image](https://github.com/bluz71/vim-moonfly-colors/assets/25379882/6c67af2c-3f6b-40a1-96e9-c675a62dc34e)
Like this for example, maybe using the same background color as for Lualine?

**Comment by bluz71** [comment]
I tried styling Bufferline as documented in [this issue](https://github.com/bluz71/vim-moonfly-colors/issues/24); followed by another go around a couple months later.

I gave up in the end both times since I found styling for this particular plugin horrendously difficult. This plugin has a many auto-generated highlight groups which were very difficult / impossible to figure out. 

In the end, the less styling I did the better the result. Maybe it has regressed since I last checked.

Basically, I am no longer willing to invest the time to sort out Bufferline issues.

PR is welcome.
Alternatively, [Barbar](https://github.com/romgrk/barbar.nvim) should look better.

I probably should remove mention that Bufferline is styled from the README since I am not willing to support it.

Sorry, I am not a fan of Bufferline.

Closing.

P.S. If you have a fix I will merge it.

**Comment by huyhoang8398** [comment]
<img width="1792" alt="Screenshot 2023-08-03 at 22 28 13" src="https://github.com/bluz71/vim-moonfly-colors/assets/25379882/ab5b780e-dbb4-4776-a23d-767273f72803">
thanks for suggesting barbar, it's working perfect and well documentation

**Comment by bluz71** [comment]
Barbar has understandable highlight groups from my perspective, so I do fully support it.

I am very pleased to hear that it works well for you, good result in the end.

Best regards.
